vars.go

```diff
@@ -8,274 +8,470 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+func newMetric(subsys string, name string, descr string) *prometheus.Desc {
```

Comment:
newMetric -> newDesc ? 

If the argument is of the same type you only need to specify it once: 
``subsys, name, descr string``